tests/decorator/test_methods.py

```diff
@@ -38,8 +48,7 @@ def test_defaults(self):
         self.assertFalse(ACL_METHODS in self.get('/defaults', origin='www.example.com').headers)
         self.assertFalse(ACL_METHODS in self.head('/defaults', origin='www.example.com').headers)
         res = self.preflight('/defaults', 'POST', origin='www.example.com')
-        for method in ALL_METHODS:
-            self.assertTrue(method in res.headers.get(ACL_METHODS))
+        self.assertIsNone(res.headers.get(ACL_METHODS))
```

Comment:
This is backward incompatible. Earlier if the cross_origin decorator had no methods specified all the methods from the default or app level configuration would be used but now the methods will be restricted to what the view supports.